content/docs/guides/troubleshooting/container_lifecycle.md

```diff
@@ -21,7 +21,21 @@ It is recommended that application containers be resilient enough to the initial
 
 It is important to note that the [container's restart policy](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#restart-policy) also influences the startup of application containers. If an application container's startup policy is set to `Never` and it depends on network connectivity to be ready at startup time, it is possible the container fails to access the network until the Envoy proxy sidecar is ready to allow the application container access to the network, thereby resulting in the application container to exit and never recover from a failed startup. For this reason, it is recommended not to use a container restart policy of `Never` if your application container depends on network connectivity at startup.
 
-### Related issues (work in progress)
+## When the application container is designed to run to completion
+
+Application containers like those running in Kubernetes Jobs that are meant to perform a certain set of tasks and then exit may not appear to have finished based on the Job's status because the Envoy sidecar container runs indefinitely. For the Pods controlled by Jobs to exit as they would without the sidecar, it is recommended for the application container to signal for Envoy to exit when its tasks are complete on its `/quitquitquit` HTTP endpoint. For Jobs driven by shell scripts, this may look something like
```

Comment:
```suggestion
Application containers, like those running in [Kubernetes Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/job/), that are meant to perform a certain set of tasks and then exit may not appear to have finished based on the Job's status. This status is  expected and is due to the Envoy sidecar container running indefinitely. Update your pods controlled by Jobs to use the Envoy sidedcar's `/quitquitquit` HTTP endpoint to signal for Envoy to exit when your pod has completed its tasks. 

 For Jobs driven by shell scripts, this may look something like
```